### DISCUSSION TITLE
Hide tags in the TagPicker field

### DISCUSSION THREAD

**Discussion by ToyinY** [rsuite/rsuite#3969] [Q&A]
Hi,

Is there a way to hide the tags in the tag picker field when options are selected? I believe the check picker was replaced by the tag picker and I am wondering if there was a type of multi select drop down that did not show tags in the text field. Let me know if this is possible.

Thanks

**Comment by simonguo** [comment]
You can try using the `renderValue` property to customize how the selected values are rendered.

```tsx
const renderValue = (values, items, tags) => {
  return <span>{values.length} items selected</span> 
};

<TagPicker renderValue={renderValue} />
```
